charts/tractusx-connector-azure-vault/templates/deployment-controlplane.yaml

```diff
@@ -289,10 +289,8 @@ spec:
             ## BDRS CLIENT ##
             #################
 
-            {{- if .Values.controlplane.bdrs.server.url }}
             - name: "TX_IAM_IATP_BDRS_SERVER_URL"
               value: {{ .Values.controlplane.bdrs.server.url | required ".Values.controlplane.bdrs.server.url is required" | quote }}
-            {{- end }}
```

Comment:
@paullatzelsperger I'm not sure if this was implemented on purpose back then. Do you possibly know?

Reply:
I already [asked in matrix](https://matrix.to/#/!mYxOilDPMLCQhMoIVc:matrix.eclipse.org/$mYmAhWWFSODdIgAzVb5ezMMUJFLrWRFc8iYffr4Y2ms?via=matrix.eclipse.org&via=matrix.org&via=dev-null.rocks), but haven't received any answer

Reply:
I don't recall :(
Since the BDRS Url is mandatory anyway, I think we could remove the `if`